ui/shared/BugFiler.jsx

```diff
@@ -188,9 +189,13 @@ export class BugFilerClass extends React.Component {
       }
     }
 
+    if (currentRepo.name === 'try') {
+      summaryString = `[non-prod]  ${summaryString}`;
+    }
+
     this.state = {
       tooltipOpen: {},
-      summary: `Intermittent ${summaryString}`,
+      summary: `Intermittent [non-prod] ${summaryString}`,
```

Comment:
This looks like it would add `[non-prod]` twice for `try`.  I would suggest:

```
const  summaryPrefix = '';
// extra space intentional in 'try '
const prefix = currentRepo.name === 'try ' ? '[non-prod]' : '';
...

summary: `Intermittent {prefix}${summaryString}`,

```